src/gas/GasFeeController.ts

```diff
@@ -17,7 +17,7 @@ import {
   calculateTimeEstimate,
 } from './gas-util';
 import determineGasFeeSuggestions from './determineGasFeeSuggestions';
-import fetchGasEstimatesViaEthFeeHistory from './fetchGasEstimatesViaEthFeeHistory';
+import { fetchGasEstimatesViaEthFeeHistory } from './eip1559ApiFallback';
```

Comment:
Happy to take suggestions for how to organize this code. I went with a folder that reflected the reason for all of the files inside of that folder but an alternative would be to name that folder `fetchGasEstimatesViaEthFeeHistory` and that way we could continue to say `import fetchGasEstimatesViaEthFeeHistory from './fetchGasEstimatesViaEthFeeHistory'` here.